src/forms/CurrentIncome.js

```diff
@@ -130,11 +144,20 @@ const IncomeForm = function ({ current, time, setClientProperty }) {
 * @returns Component
 */
 // `props` is a cloned version of the original props. References broken.
+/** @todo  Make currentIncomeStep compatible with CurrentBenefitsStep? 
+ *         currentBenefits looks like this:
+ *         const CurrentBenefitsStep = ({ changeClient, nextStep, client, snippets })
+ * Which one is correct?
+*/
```

Comment:
Either way works. The "destructuring" style in `CurrentBenefitsStep` (which assigns each field of the `props` object to a variable immediately) is nicer since it's more explicit, so we should probably switch to that at some point, but it doesn't have to be in this PR.